docs(dedibox): Updated note about scaleway console migration MTA-7242 (#6609)
scaleway/docs-content: macros/bare-metal/dedibox-scaleway-migration.mdx

---
macro: dedibox-scaleway-migration
---

<Message type="note">

>   You can now manage resources from both the Scaleway and the Dedibox consoles. However, ordering Dedibox servers from the Scaleway console is only possible if you added a SEPA payment method in your **[Dedibox console](https://console.online.net)**. If you do not complete this step, you can only use the Dedibox console to place orders (see the [Dedibox documentation](/dedibox/quickstart/)).

Reviewer: ```suggestion
  You can now manage resources from both the Scaleway and the Dedibox consoles. However, ordering Dedibox servers from the Scaleway console is only possible if you have added a SEPA payment method in your **[Dedibox console](https://console.online.net)**. If you do not complete this step, you can only use the Dedibox console to place orders (see the [Dedibox documentation](/dedibox/quickstart/)).
```

>   You can now manage resources from both the Scaleway and the Dedibox consoles. However, ordering Dedibox servers from the Scaleway console is only possible if you have added a SEPA payment method in your **[Dedibox console](https://console.online.net)**. If you do not complete this step, you can only use the Dedibox console to place orders (see the [Dedibox documentation](/dedibox/quickstart/)).

Reviewer 2: ```suggestion
  You can manage resources from both the Scaleway and the Dedibox consoles. Ordering Dedibox servers from the Scaleway console is only possible if you have added a SEPA payment method in your **[Dedibox console](https://console.online.net)**. Otherwise, you can only use the Dedibox console to place orders (see the [Dedibox documentation](/dedibox/quickstart/)).
```

Author: Hello @Loic-kd ; ta version est beaucoup plus efficace et serait normalement ce vers quoi je pencherais (mes premiers drafts étaient très similaires), mais la mise à jour est liée à une vague de tickets supports ce qui nous a fait opter pour l'approche la plus didactique. Le but étant que Paul challenge tout ça à son arrivée. 
Merci beaucoup pour ton retour 🙌
